hijing.sh

```diff
@@ -1,7 +1,7 @@
 # a pythia6 recipe based on the one from FairROOT
 package: hijing
 version: "%(tag_basename)s"
-tag: "v1.36"
+tag: master
```

Comment:
Please tag it.

Reply:
hi @ktf , can I tag it myself?
I can tag in on my fork, but how do I port the tag in the `alisw` repo?